src/libs/actions/SidePane.ts

```diff
@@ -0,0 +1,33 @@
+import type {OnyxMergeInput} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+type Options = {
+    /** Whether to update the narrow layout along with the large screen layout */
+    shouldUpdateNarrowLayout?: boolean;
+
+    /** Whether to update only the narrow layout without affecting the large screen layout */
+    shouldOnlyUpdateNarrowLayout?: boolean;
+};
+
+/**
+ * Updates the side pane state in Onyx.
+ *
+ * @param isOpen - Determines whether the side pane should be open or closed.
+ * @param [options] - Additional options for updating the layout.
+ */
+function triggerSidePane(isOpen: boolean, {shouldUpdateNarrowLayout = false, shouldOnlyUpdateNarrowLayout = false}: Options = {}) {
+    const value: OnyxMergeInput<typeof ONYXKEYS.NVP_SIDE_PANE> = {};
+
+    if (!shouldOnlyUpdateNarrowLayout) {
+        value.open = isOpen;
+    }
+    if (shouldUpdateNarrowLayout || shouldOnlyUpdateNarrowLayout) {
+        value.openNarrowScreen = isOpen;
+    }
```

Comment:
NAB - If you open/close the pane on mobile and also have the app open desktop at the same time, wouldn't it cause the desktop app to switch to the narrow screen version, since the NVP would be also updated in Onyx in the desktop client?

Reply:
Once it's connected with backend NVP value I'll check it out 👌